src/tui-editor.service.ts

```diff
@@ -5,12 +5,21 @@ export class TuiService {
   editor: any;
   constructor() { }
   createEditor(options: object): any {
-    this.editor = new TuiEditor(Object.assign({
-      el: document.querySelector('.ngx-tui-editor'),
-      initialEditType: 'markdown',
-      previewStyle: 'vertical',
-      height: '300px'
-    }, options));
+    if (options.viewer) {
```

Comment:
When you say: "Can't load Tui-viewer" - What error are you seeing exactly that this is addressing?

Reply:
Thanks for the reply!
Changed to any type to solve # 7.  When I used it, I have to download the package and modify it with any type.

Can not load The Tui-viewer does not tell you the error message.
The Tui-editor documentation provided by nhn uses the 'static factory method' when using the viewer. I thought I should be able to use the viewer by adding the 'viewer: true' value to the option object while using the ngx-tui-editor.

I told you before, but my English skills are immature. If I misunderstood the intent of the question, can you explain it in more detail?

Reply:
Your English is great! This answers my question. Looks good to me, thanks for the contribution!